example/ghost/t8_test_ghost_large_level_diff.cxx

```diff
@@ -103,10 +103,14 @@ t8_ghost_large_level_diff(const char* path, int dim, int level, int refine,
     sc_flopinfo_t       fi, snapshot;
     sc_statinfo_t       stats[1];
 
-    //T8_ASSERT(path == NULL);
-    //get cmesh
-    //cmesh = t8_cmesh_from_msh_file((char *) path, 1, comm, dim, 0);
-    cmesh = t8_cmesh_new_hypercube(T8_ECLASS_PRISM, comm, 0, 0, 0);
+
+    if(path != NULL){
+        cmesh = t8_cmesh_from_msh_file((char *) path, 1, comm, dim, 0);
```

Comment:
the (char*) cast is wrong since t8_cmesh_from_msh_file expects a const char* (which path is already)